server/service/jct/api.coffee

```diff
@@ -215,6 +220,75 @@ API.service.jct.suggest.funder = (str, from, size) ->
     res.push({title: f.funder, id: f.id}) if matches
   return total: res.length, data: res
 
+API.service.jct.suggest.iac = (str, from, size) ->
+  _gather_rec = (data) ->
+    rec = {
+      'id': data.ror,
+      'title': data.title,
+      'country': data.country ? '',
+      'ror': data.ror
```

Comment:
This could do with having the acronym in an `alternate` field as that's what the UI is expecting

Reply:
or if there's an alias, put that in `alternate` instead